src/libraries/Microsoft.PowerFx.Core/Functions/UserDefinedFunction.cs

```diff
@@ -43,6 +43,8 @@ internal class UserDefinedFunction : TexlFunction
 
         public override bool IsSelfContained => !_isImperative;
 
+        public IEnumerable<UDFArg> Args => _args;
```

Comment:
>Args [](http://example.com/codeflow?start=35&length=4)

Do we need this? I couldn't find a reference to it

Reply:
Oh yeah you're right. Left in by mistake from a stale local change